content/desktop/networking.md

```diff
@@ -53,6 +53,22 @@ See:
 - [Proxies on Mac](settings/mac.md#proxies)
 - [Proxies on Windows](settings/windows.md#proxies)
 
+### SOCKS5 proxy support (Beta)
+
+SOCKS (Socket Secure) is a protocol that facilitates the routing of network packets between a client and a server through a proxy server. It provides a way to enhance privacy, security, and network performance for users and applications.
+
+With Docker Desktop 4.28 and later, you can enable SOCKS proxy support to allow outgoing requests, such as pulling images, and access Linux container backend IPs from the host. 
+
+To enable and set up SOCKS proxy support:
+
+1. Navigate to **Settings** in Docker Desktop. 
+2. Go to **Features in development** and then on the **Beta** tab check the **Enable SOCKS proxy support** option. 
```

Comment:
**Question** I'm on DD 28.0 and I see this option: `Enable SOCKS proxy for accessing Linux containers` - with this release will this wording match this doc or is this a different check box?